configs/envoy_double_proxy.template.yaml

```diff
@@ -0,0 +1,110 @@
+---
+{% macro listener(address,ssl,proxy_proto) -%}
+- address: "{{ address }}"
+  {% if ssl -%}
+  ssl_context:
+    alpn_protocols: h2,http/1.1
+    alt_alpn_protocols: http/1.1
+    cert_chain_file: certs/servercert.pem
+    private_key_file: certs/serverkey.pem
+  {% endif -%}
+  {% if proxy_proto -%}
+  use_proxy_proto: true
+  {% endif -%}
+  filters:
+  - name: http_connection_manager
+    config:
+      codec_type: auto
+      tracing:
+        operation_name: ingress
+      idle_timeout_s: 840
+      access_log:
+      - path: "/var/log/envoy/access_error.log"
+        filter:
+          type: logical_or
+          filters:
+          - type: status_code
+            op: ">="
+            value: 500
+          - type: duration
+            op: ">="
+            value: 1000
+          - type: traceable_request
+      - path: "/var/log/envoy/access.log"
+      stat_prefix: router
+      {% if proxy_proto -%}
+      use_remote_address: true
+      {% endif -%}
+      route_config:
+        virtual_hosts:
+        - name: all
+          domains:
+          - "*"
+          routes:
```

Comment:
the organization of the route object is different: https://www.envoyproxy.io/docs/envoy/latest/api-v2/api/v2/route/route.proto#envoy-api-msg-route-route